src/frontends/onnx/frontend/src/core/node.cpp

```diff
@@ -339,6 +339,11 @@ std::vector<Graph> Node::get_attribute_value(const std::string& name, std::vecto
     return m_pimpl->template get_attribute_value<std::vector<Graph>>(name, std::move(default_value));
 }
 
+template <>
+ONNX_IMPORTER_API bool Node::get_attribute_value(const std::string& name, bool default_value) const {
```

Comment:
I know that `PriorBox` is not an op from ONNX domain, but based on https://github.com/onnx/onnx/blob/master/onnx/onnx.proto#L118 attribute can't be bool type.
In ONNX domain ops, there is a practice to use 0/1 ints to represent bools.
@tomdol What do you think?
At least it will be nice to have a test that uses bool as an attribute.

Reply:
Good catch, I completely forgot about it. Implicit conversion from int retrieved from the model will do.

Reply:
@mbencer  @tomdol  Fixed 😊